### PR TITLE
Fixed #2097 - twitter icon url modified

### LINKED ISSUE #2097: twitter icon to https://twitter.com/webcompat/with_replies
The https://twitter.com/webcompat/with_replies doesn't work anymore for the last couple of months if you are not logged in. This is a bummer. Intended change by twitter.

The only equivalent now is something along
https://twitter.com/search?f=tweets&vertical=default&q=%40webcompat&src=typd

### PR DESCRIPTION
**Pull request by amitsin6h**
Changed the twitter icon url from the footer as the url didn't worked if not logged in.

### CODE REVIEW

File: webcompat/templates/shared/footer.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
       </ul>
       <ul class="wc-Footer-list wc-Footer-list--right r-ResetList">
         <li class="wc-Footer-item wc-Footer-item--follow">
-          <a class="wc-Footer-link" href="https://twitter.com/webcompat/with_replies" target="_blank">
+          <a class="wc-Footer-link" href=" https://twitter.com/search?f=tweets&vertical=default&q=%40webcompat&src=typd" target="_blank">
```
Comment (zoepage) [inline webcompat/templates/shared/footer.html]: could you please remove the empty space in the url? thanks!

**Comment by zoepage** [comment]
Just one small thing. Could you also please change the commit message so it fits our conventions like this: `Issue #2097 - twitter icon url modified`.

You can do this with `git commit --amend` 
More info can be found here: https://www.atlassian.com/git/tutorials/rewriting-history

Thanks!

**Comment by amitsin6h** [comment]
@zoepage could you please review it.

**Comment by zoepage** [comment]
The PR looks good. Could you please change the commit messages  `Twitter icon url modified` and `Removed empty space` as described above? Feel free to squash them into one commit.

Thanks!

**Comment by amitsin6h** [comment]
@zoepage hope now its ready to merged.

**Comment by zoepage** [comment]
Sorry, looks like we are misunderstanding each other. :)

You basically would just need to run following:
`git commit --amend -m "Issue #2097 - twitter icon url modified"`

We are almost there!

**Comment by amitsin6h** [comment]
@zoepage please review it now.

**Pull request merged**